src/main/java/com/dataloom/neuron/Receptor.java

```diff
@@ -19,6 +19,8 @@
 
 package com.dataloom.neuron;
 
+import com.dataloom.neuron.signals.Signal;
+
 public interface Receptor<T extends Signal> {
```

Comment:
I don't have a great reason, but the initial thought process was that `Signal` would be a base class, and different services would have their own `Receptor`s that process their own specific `Signal`s.